packages/app/src/common/hooks/use-message-pong.ts

```diff
@@ -1,28 +1,26 @@
 import { useEffect, useCallback } from 'react';
-import { useSelector } from 'react-redux';
-import { selectAuthRequest } from '@store/onboarding/selectors';
+// import { useSelector } from 'react-redux';
+// import { selectAuthRequest } from '@store/onboarding/selectors';
 import { getEventSourceWindow } from '@common/utils';
 
 export const useMessagePong = () => {
-  const authRequest = useSelector(selectAuthRequest);
-  const sendMessage = useCallback(
-    (event: MessageEvent) => {
-      if (event.data.method === 'ping') {
-        const source = getEventSourceWindow(event);
-        source?.postMessage(
-          {
-            method: 'pong',
-            authRequest: event.data.authRequest,
-            source: 'blockstack-app',
-          },
-          event.origin
-        );
-      }
-    },
-    [authRequest]
-  );
+  const sendMessage = useCallback((event: MessageEvent) => {
+    if (event.data.method === 'ping') {
+      console.log('ping!');
```

Comment:
would be great to have this env dependent. eg don't console log so much in the built versions

Reply:
Yeah there could definitely be strong improvements by logging more in dev mode. This one can just be removed, it was for debugging and is not needed anymore.